<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>160</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nhap n :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>190</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Kiem tra snt :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>220</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Tim snt nho hon n :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_n">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>160</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_kt">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nho">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_nhap">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>93</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Nhap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_thoat">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>160</y>
      <width>93</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Thoat</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
